tests/test_selenium.py

```diff
@@ -54,30 +69,85 @@ def test_login():
             script = polymer_is_trash(['main-page', 'dashboard-view', '#Quick-Links'])
             wait.until(lambda x: x.execute_script(script))
 
-            # Navigate to Jupyter frontend
-            script = polymer_is_trash(
-                ['main-page', 'iframe-link[href=\'/jupyter/?ns=admin\'] .menu-item']
-            )
-            wait.until(lambda x: x.execute_script(script))
-            driver.execute_script(script + '.click()')
-
-            # Click "New Server" button
-            script = polymer_is_trash(['main-page', 'iframe-container', 'iframe'])
-            script += ".contentWindow.document.body.querySelector('#add-nb')"
-            wait.until(lambda x: x.execute_script(script))
-            driver.execute_script(script + '.click()')
-
-            # Enter server name
-            script = polymer_is_trash(['main-page', 'iframe-container', 'iframe'])
-            script += ".contentWindow.document.body.querySelector('#mat-input-0')"
-            wait.until(lambda x: x.execute_script(script))
-            driver.execute_script(script + '.value = "foobar"')
-            driver.execute_script(script + '.dispatchEvent(new Event("input"))')
-
-            # Submit form
-            script = polymer_is_trash(['main-page', 'iframe-container', 'iframe'])
-            script += ".contentWindow.document.body.querySelector('form').submit()"
-            driver.execute_script(script)
+            yield driver, wait, url
         except Exception:
             driver.get_screenshot_as_file('/tmp/selenium.png')
             raise
```

Comment:
I'm confused why this doesn't seem to be working on the AWS timeouts.

Reply:
Fixed. At that point, there's actually no exception being raised. I assume pytest is doing some magic in there